octodns/provider/ns1.py

```diff
@@ -298,6 +309,34 @@ def __init__(self, id, api_key, retry_count=4, monitor_regions=None,
         self._client = Ns1Client(api_key, parallelism, retry_count,
                                  client_config)
 
+    # Allowed filter configurations:
+    # 1. Filter chain is the basic filter chain
+    # 2. In addition, it could have the 'geofence_country' filter and or
+    # 'geofence_regional' filter only at index 1 in the filter chain
+    # 3. If both country and regional filters are present, they are required
+    # to be as below:
+    #   - 'geofence_regional' at index 1
+    #   - 'geofence_country' at index 2
+    # Any other deviation is returned as an unsupported filter configuration
+    def _valid_filter_config(self, filter_cfg):
+        has_region = 'geofence_regional' in [f['filter'] for f in filter_cfg]
```

Comment:
Should likely be able to do 

```suggestion
        has_region = self._REGION_FILTER in filter_cfg
```

Reply:
Fixed.